src/components/CytoscapeGraph/CytoscapeReactWrapper.tsx

```diff
@@ -146,27 +159,82 @@ export class CytoscapeReactWrapper extends React.Component<CytoscapeReactWrapper
     }
   }
 
+  createMenuOption(link: string, title: string, target: string = '_self') {
+    const listElement = document.createElement('li');
+    listElement.setAttribute('class', 'kiali-graph-context-menu-item');
+    listElement.onclick = CytoscapeReactWrapper.redirectContextLink;
+    listElement.innerHTML = `<a class='kiali-graph-context-menu-item-link' target="${target}" href="${link}" >${title}</a>`;
+    return listElement;
+  }
+
   private makeTippyForNode = (node: any) => {
     return tippy(node.popperRef(), {
       content: (() => {
         // Remember we on Tippy.js which is html over the cytoscape canvas -- not React JSX
         // @todo: Refactor this to ReactDOM.render() to use JSX
         const tippyDiv = document.createElement('div');
-        tippyDiv.setAttribute('class', 'kiali-graph-context-menu-container');
+        tippyDiv.setAttribute('class', 'popover-content');
```

Comment:
Where is this style defined?

If you replace the style, perhaps you want to remove the `kiali-graph-context-menu-container` in CytoscapeGraph.scss.

Reply:
this is an eror thanks @israel-hdez I was triying to overwrite the theme of tippyDiv because I didn't know why CutoscapeGraph.scss didn't work, after that i added the css file to the index styles. I'll remove it

Reply:
FWIW, I'm working on adding support to use react Components instead of raw html code.